.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "rust/lb-operator/vendor/csi"]
```

Comment:
This doesnt exist anymore?

Reply:
Interesting.. I think this is just a name that doesn't actually mean anything (compared to `path` right below, anyway). Updated it anyway in 73af0e3c130bb586a1fab32dd8e796c53b48045e.